### DISCUSSION TITLE
Frage zu MQTT

### DISCUSSION THREAD

**Discussion by sirius-black-2** [camueller/SmartApplianceEnabler#574] [Allgemein]
Hallo,

ich versuche gerade meine Tasmota Geräte von HTTP auf MQTT Zähler umzustellen, doch so ganz komme ich mit der Anleitung nicht klar...Mosquitto wurde bei der automatisierten Installtion korrekt installiert und die Parameter in den Dateien (Persistence false ...) passen. Im Tasmota Menu wurde das Häkchen bei MQTT aktiviert. Bei den MQTT Einstellungen steht der richtige Port (1883) an den Feldern Client und Benutzer habe ich nichts verändert, der Topic wurde "Arbeitszimmer" benannt. Bei Host habe ich die Raspi IP im lokalen Netzwerk eingetragen (also 192.168.usw) . 
Im SAE steht unter Topic "tele/tasmota/SENSOR", wie in der Anleitung oder muss hier der Topic aus dem Tasmota rein also in meinem Fall "Arbeitszimmer", die restlichen Parameter sind aus der Anleitung und mMn klar (Zählerstand, JSON; $.ENERGY.Total; $.Time). Vielleicht kann mir ja jemand auf die Sprünge helfen, Danke im vorraus.
Noch eine Frage zum go E-Charger: Der November ist ja ein sehr dunkler Monat und dementsprechend wenig PV Ertrag kommt momentan runter. Wenn man die Option "PV Überschuß" wählt, wie weit kann der SAE die Wallbox den drosseln? In der Go E-Charger APP ist minimal 6 A möglich (was ca. 4 KW entspricht) 
Und eine letze Frage, weiß jemand wie man an das SOC Script beim Opel Mokka kommt?

Danke an die Community

**Comment by camueller** [comment]
> Im SAE steht unter Topic "tele/tasmota/SENSOR", wie in der Anleitung oder muss hier der Topic aus dem Tasmota rein also in meinem Fall "Arbeitszimmer"

Das sollte eigentlich klar sein, dass das Topic bei Sender und Empfänger gleich sein müssen :-)
Bei meinem Kühlschrank ist es z.B. `tele/kuehlschrank/SENSOR`.

> Noch eine Frage zum go E-Charger: Der November ist ja ein sehr dunkler Monat und dementsprechend wenig PV Ertrag kommt momentan runter. Wenn man die Option "PV Überschuß" wählt, wie weit kann der SAE die Wallbox den drosseln? In der Go E-Charger APP ist minimal 6 A möglich (was ca. 4 KW entspricht)

6A ist Minimum, aber Überschussladen sollte man generell nur einphasig machen. Das ergibt dann ca. 1380W, die so auch als min. Leistungsaufnahme in den Geräteeigenschafter der Wallbox im SAE eingetragen werden sollten.

**Reply by sirius-black-2** [reply]
Ah, vielen Dank für die Erklärung! Werd ich gleich testen.
